zha/application/platforms/sensor/__init__.py

```diff
@@ -658,7 +654,7 @@ def formatter(self, value: int) -> int | float:
 
         value = float(value * multiplier) / divisor
         if value < 100 and divisor > 1:
-            return round(value, self._decimals)
+            return round(value, self._attr_suggested_display_precision)
```

Comment:
For the `ElectricalMeasurement` sensor, we effectively set:
- `_attr_suggested_display_precision = 1` if `divisor > 1 and value < 100`
- `_attr_suggested_display_precision = 0` otherwise

Home Assistant caches `suggested_display_precision` so I don't think we can dynamically change it.

Should we globally set the decimals to 1?

CC @TheJulianJES @dmulcahey